python/lsst/jointcal/jointcal.py

```diff
@@ -381,11 +392,11 @@ def run(self, dataRefs, profile_jointcal=False):
         with pipeBase.cmdLineTask.profile(load_cat_prof_file):
             self._write_results(associations, astrometry.model, photometry.model, visit_ccd_to_dataRef)
 
-        lsst.verify.output.output_quantities("jointcal", self.metrics)
+        self.job.write(self.config.metricOutputFile)
 
         return pipeBase.Struct(dataRefs=dataRefs,
                                oldWcsList=oldWcsList,
-                               metrics=self.metrics,
+                               measurements=self.job.measurements,
```

Comment:
If we're writing the output to some obvious place, we shouldn't need to return it in the Struct anymore, and just have the tests load that file.

Reply:
Good point.  I'll look into that.

Reply:
This actually got a little ugly.  I think I may prefer to just pass it back through `result`, but see what you think.

Reply:
I actually need to return the job anyway because I'm using the `Butler` in the `TaskRunner` to write the output.  I'm going to revert to just using the job that is passed back to run the tests.  Let me know if you think this is a showstopper.